warehouse/query-core/src/main/java/datawave/query/tracking/ActiveQueryLog.java

```diff
@@ -100,10 +100,9 @@ public static ActiveQueryLog getInstance(String name) {
         }
         
         // If no log currently exists for the name, create one.
```

Comment:
This should go back to the way it was before the last commit so that we check for the named AQL in the logCache and add it if necessary all inside the synchronized block

Reply:
Fixed.